packages/jellyfish-testing/src/icxorderbook.ts

```diff
@@ -0,0 +1,243 @@
+import { Testing } from './index'
+import BigNumber from 'bignumber.js'
+import { icxorderbook } from '@defichain/jellyfish-api-core'
+
+export class TestingICX {
+  public symbolDFI: string
+  public symbolBTC: string
+  public accountDFI: string
+  public accountBTC: string
+  public poolOwner: string
+  public idDFI: string
+  public idBTC: string
+  public ICX_TAKERFEE_PER_BTC: BigNumber
+  public DEX_DFI_PER_BTC_RATE: BigNumber
+
+  constructor (
+    private readonly testing: Testing
+  ) {
+    this.accountDFI = ''
+    this.accountBTC = ''
+    this.poolOwner = ''
+    this.idDFI = ''
+    this.idBTC = ''
+    this.ICX_TAKERFEE_PER_BTC = new BigNumber(0)
+    this.DEX_DFI_PER_BTC_RATE = new BigNumber(0)
+    this.symbolDFI = 'DFI'
+    this.symbolBTC = 'BTC'
+  }
+
+  async setup (): Promise<void> {
+    await this.createAccounts()
+    await this.createBTCToken()
+    await this.initializeTokensIds()
+    await this.mintBTCtoken(100)
+    await this.fundAccount(this.accountDFI, this.symbolDFI, 500)
+    await this.fundAccount(this.accountBTC, this.symbolDFI, 10) // for fee
+    await this.createBTCDFIPool()
```

Comment:
```suggestion
```

A large part of the setup can be replaced with:
```ts
const pair = await testing.fixture.createPoolPair({
  a: { amount: '500', symbol: 'DFI' },
  b: { amount: '100', symbol: 'BTC' }
})
await testing.icxorderbook.setTakerFee(0.001)
```

Maybe we could just use this instead, since it's cleaner contextually what you are setting up also.

Reply:
Contextually the only setup you are doing here is setting TakerFee, other than that it's creating the pool pair and we have an existing fixture for that.

Reply:
I think we still need setup() in order to set 
```
    this.accountDFI = ''
    this.accountBTC = ''
    this.poolOwner = ''
    this.idDFI = ''
    this.idBTC = ''
    this.ICX_TAKERFEE_PER_BTC = new BigNumber(0)
    this.DEX_DFI_PER_BTC_RATE = new BigNumber(0)
```